cypress/support/commands.js

```diff
@@ -30,33 +30,27 @@ Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy^="${selector}"]`);
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
-  cy.request('POST', '/api/users', {
-    user: {
-      email,
-      username,
-      password
-    }
-  });
+Cypress.Commands.add('register', ({ email, username, password }) => {
+  cy.request('POST', '/users', {
+    email,
+    username,
+    password
+  }).then((res) => ({
+    ...res.body.user,
+    password
+  }));
 });
 
-Cypress.Commands.add('login', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
-  cy.request('POST', '/api/users', {
-    user: {
-      email,
-      username,
-      password
-    }
-  }).then(response => {
-    const user = {
-      bio: response.body.user.bio,
-      effectiveImage: "https://static.productionready.io/images/smiley-cyrus.jpg",
-      email: response.body.user.email,
-      image: response.body.user.image,
-      token: response.body.user.token,
-      username: response.body.user.username,
-    };
-    window.localStorage.setItem('user', JSON.stringify(user));
-    cy.setCookie('auth', response.body.user.token);
-  });
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+  cy.get(`[placeholder="${placeholder}"]`);
+});
+
+Cypress.Commands.add('login', (user) => {
+  cy.register(user)
+    .then(({ email, password }) => {
+      cy.request('POST', '/users/login', { user: { email, password } })
+        .then((res) => {
+          cy.setCookie('drash_sess', res.body.user.token);
+        });
+    });
 });
```

Comment:
The 'login' command here overwrites the previously defined 'login' command on line 14. This could lead to confusion or unexpected behavior if both commands are intended to be used. Consider renaming one of the commands to avoid this conflict.